src/scss/components/_listings.scss

```diff
@@ -90,179 +103,168 @@ $listing-borderRadius       : 4px;
                 border-left: 1px solid $listing-borderColor;
-
-                .c-listing--offline & {
-                    border-color: $listing-offlineBorderColor;
-                }
             }
         }
 
-        .c-listing-item-rating {
-            padding: 2px 0 3px;
+        .c-listing-item-detailsRow {
+            display : flex;
+            align-items : center;
+            margin-bottom: spacing() / 2;
+        }
 
-            .c-listing-item-ratingStars,
-            .c-listing-item-ratingText {
-                vertical-align: middle;
-            }
+        .c-listing-item-detailsRow-icon {
+            width: 20px;
+            height: 20px;
+            fill: $grey--dark;
         }
 
-        .c-listing-item-details {
-            width: 100%;
-            padding-left: 56px;
+        .c-listing-item-detailsRow-text {
+            margin-left: spacing();
+        }
+
+        .c-listing-item-promotion {
 
-            @include media('>=tiny') {
-                padding-left: 66px;
+            svg {
+                fill: $listing-promotionColor;
             }
 
-            @include media('>wide') {
-                right: 0;
-                width: 260px;
-                margin-top: 10px;
-                position: absolute;
-                bottom: spacing(x2);
-                padding-left: spacing(x2);
+            .c-listing-item-promotionText {
+                color: $listing-promotionColor;
             }
         }
 
-        .c-listing-item-distance {
-            position: absolute;
-            right: spacing(x2);
-            bottom: spacing(x2);
-
-            @include media('>mid') {
-                right: 0;
-            }
+        .c-listing-item-title {
+            word-break: break-word;
+            padding-right: spacing();
+            font-weight: $font-weight-base;
 
             @include media('>wide') {
-                position: static;
-                line-height: 0.8;
-                padding-left: 80px;
-                margin-bottom: spacing() / 2;
+                margin: (spacing() / 2) 0;
+                padding-right: spacing(x2);
             }
         }
 
-        .c-listing-item-distanceIcon {
-            //TODO - remove target / styling of SVG when inline-helper allows us to add class
-            svg {
-                width: 10px;
-                height: 10px;
-                fill: $listing-iconFill;
-            }
+        .c-listing-item-new {
+            color: $green;
+            top: -2px;
+            position: relative;
+            margin-right: spacing() / 2;
+        }
+
+        .c-listing-item-rating {
+            padding: 2px 0 3px;
+        }
+
+        .c-listing-item-ratingText {
+            top: -3px;
+            position: relative;
+            margin-left: spacing() / 2;
+            @include font-size(small, true);
         }
 
         .c-listing-item-text {
             margin-top: 0;
         }
 
+        .c-listing-item-textInline {
```

Comment:
This looks like it may be a modifier.  How is this used in markup?

Reply:
removed - used the layout class `l-inline` instead 👍